src/core/dom/form.js

```diff
@@ -3,7 +3,9 @@ import {ancestorElementsByTag} from '#core/dom/query';
 import {toArray} from '#core/types/array';
 import {dict} from '#core/types/object';
 
-/** @const {string} */
+/**
+ * @const
```

Comment:
Can skip both the `const` and the `@type` here

Reply:
This was removed after you opened the diff but before you left the comment 🙃